.github/workflows/publish-npm-packages.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         with:
           check-latest: true
           registry-url: https://registry.npmjs.org
-          node-version: '16'
+          node-version: lts/*
```

Comment:
Esther noticed that GHA still caches `lts == 14`, which is why we explicitly set it to '16' for now

Reply:
Good catch. I've explicitly set [`check-latest`](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#check-latest-version) to true so the installer looks up the [actual latest LTS version](https://github.com/actions/node-versions/releases). Does this address the original concern?

Reply:
You tell me :)

Reply:
Should work but no more caching benefit.

Reply:
The one workflow where `check-latest` was already set to true (before this PR) [is indeed using](https://github.com/ampproject/amphtml/runs/4091688239?check_suite_focus=true#step:3:3) the correct latest version of nodejs.

![image](https://user-images.githubusercontent.com/26553114/140173610-a4078e16-cfe2-424a-9673-cd0f6d99aab2.png)

Based on this, I believe the combination of `lts/*` and `check-latest` guarantees that the latest LTS is used.

Reply:
> Should work but no more caching benefit.

The source code at https://github.com/actions/setup-node/blob/4d0182af5ead0b7bf53e87e49aa959476f5501d3/src/installer.ts#L52-L79 suggests that while `check-latest` always fetches the latest version, it downloads the node installation binaries only when there's a cache miss. Seems like most of the benefit is retained.

Reply:
Turns out `check-latest` is a lie. Maybe the `lts/*` alias isn't  working like https://github.com/actions/setup-node/issues/26#issuecomment-871413720 is suggesting?

![image](https://user-images.githubusercontent.com/26553114/140177357-7d2de489-5582-437c-a2d7-8e4acd8957de.png)

Will poke around some more and if all else fails, I'll restore the hard-coded `'16'`.